src/constants.ts

```diff
@@ -60,4 +60,6 @@ export const AzureClouds = {
 
 export const CommentIdentifiersRegex: RegExp = /^\s*(#|\/{2})/;
 
-export const VariableDefinitionRegex: RegExp = /^\s*@([^\s=]+)\s*=\s*(.+)\s*$/;
\ No newline at end of file
+export const VariableDefinitionRegex: RegExp = /^\s*@([^\s=]+)\s*=\s*(.+)\s*$/;
+
+export const RequestVariableDefinitionRegex: RegExp = /#.*\s+@name\s+(\w+).*/;
```

Comment:
I think the request variable definition should occupy a whole line, so I think the syntax may be like `^\s*#{1,}\s+@name\s+(\S+)\s*$`. One important difference between our proposals is that no non whitespace characters are allowed between __#__ and __@name__

Reply:
I'm unsure about this change now. `\S` seems a bit too broad, and will allow characters that make the regex even more complex.